app/controllers/resources_controller.rb

```diff
@@ -15,7 +15,10 @@ def create
     end
 
     result = BackgroundJobResult.create
-    IngestJob.perform_later(druid: response[:pid], background_job_result: result)
+
+    IngestJob.perform_later(druid: response[:pid],
```

Comment:
I don't see a test for the IngestJob invocation.

Reply:
Okay, added one.

Reply:
I don't see one in the branch yet FWIW!

Reply:
oops, forgot to push.